GOVERNANCE.md

```diff
@@ -0,0 +1,124 @@
+work together to achieve the project's goals:
+
+* A Board of Directors
+* A Technical Steering Committee
+
+### Board of Directors
+
+The Diamond Bluff project Board of Directors shall be composed as follows:
+
+* One representative from each Platinum member.
+* One Silver member representative elected annually by the Silver members for
+  every 5 Silver members, with at least one representative, and no more than 3.
+
+The Board of Directors shall have the following responsibilities:
+
+* Approve a budget directing the use of funds raised from all sources of
+  revenue.
+* Elect a Chair of the Diamond Bluff project to preside over meetings,
+  authorize expenditures approved by the budget, and manage any day-to-day
+  operations, and vote on decisions or matters before the Board of Directors.
+
+The Board of Directors shall::
+
+* Demonstrate plans and the means to coordinate with the open source project’s
+  developer community, including on topics such as branding, logos, and other
+  collateral that will represent the community.
+* Engage in a professional manner consistent with maintaining a cohesive
+  community, while also maintaining the goodwill and esteem of the project's
+  foundation, in the open source software community.
+  community.
+* Respect the rights of all trademark owners, including any branding and usage
+  guidelines.
+* Engage the project's foundation for all press and analyst relations activities.
+* Upon request, provide information regarding project participation, including
+  information regarding attendance at project-sponsored events, to the
+  project's foundation.
+* Engage the project's foundation for any Diamond Bluff project websites, and
+  operate under such rules and procedures as may from time to time be approved
+  by the Diamond Bluff Board of Directors and confirmed by the project's
+  foundation.
+
+### Technical Steering Committee
+
+The Diamond Bluff Technical Steering Committee shall be composed as follows:
+
+* One appointed representative from each Platinum member
+* A maintainer from each Diamond Bluff sub-project
+
+In addition, anyone can participate on the Technical Steering Committee by
+contributing to the technical community and becoming a maintainer of a
+sub-project.
+
+The responsibilities of the Diamond Bluff Technical Steering Committee are
+as follows:
+
+* Coordinating the technical direction of the Diamond Bluff Project
+* In coordination with the Board of Directors, approving new sub-projects for
+  creation under the Diamond Bluff organization.
+* Helping to identify maintainers of sub-projects, and ensure they have the
+  technical support necessary to be successful in their role as a maintainer.
+* Communicating with external and industry organizations concering Diamond
+  Bluff technical matters.
+* Appointing represenatives to work with other open source or open standards
+  communities.
+* Voting on technical matters relating to the code base of all sub-projects.
+
```

Comment:
Add section:

## Code of Conduct Committee

The Board will appoint or elect three or more individuals to serve as a Code Of Conduct Committee (COCC).  Committee members may also be members of the Board or the TSC, or they may be outside parties (paid or unpaid) with expertise in code of conduct (CoC) enforcement, or just members of the community.  The CoCC should be selected to represent the diversity of the community.

The CoCC will establish fair and standard processes for handling reports of violations of the CoC, as well as a system of remedies, in order to protect the well-being of the project.  The CoCC will recommend the appropriate resolution to the Board, up to and including the removal of any individual from participation, either temporarily or permanently.  Should a member of the CoCC itself be accused of a violation, that accusation will be handled by an individual appointed by the foundation.